src/main/java/org/terasology/books/rendering/nui/layers/BookScreen.java

```diff
@@ -65,56 +73,50 @@
     private static final String STATUS_READING = "Reading";
     private static final String STATUS_READ_ONLY = "Read-only";
 
-    private final Logger logger = LoggerFactory.getLogger(BookScreen.class);
-
-    @In
-    private NUIManager nuiManager;
-    @In
-    private LocalPlayer localPlayer;
-    @In
-    private ClipboardManager clipboardManager;
+    private static final Logger logger = LoggerFactory.getLogger(BookScreen.class);
+    static List<String> pages;
+    static boolean leftPageEditing = true;
+    static Binding<Integer> index;
     @In
-    private PrefabManager prefabManager;
+    private static PrefabManager prefabManager;
```

Comment:
Using the `static` keywords means that all instances of the BookScreen share the same variable - is this intended for all the fields here?